src/Microsoft.Identity.Client/Platforms/Android/AuthenticationActivity.cs

```diff
@@ -136,7 +136,15 @@ protected override void OnResume()
                 browserIntent.SetData(Uri.Parse(_requestUrl));
                 browserIntent.SetPackage(chromePackage);
                 browserIntent.AddCategory(Intent.CategoryBrowsable);
-                StartActivity(browserIntent);
+                try
+                {
+                    StartActivity(browserIntent);
+                }
+                catch (ActivityNotFoundException ex)
+                {
+                    throw new MsalClientException(MsalClientException.ChromeDisabledError,
+                        "Chrome is disabled on the device, cannot proceed with authentication " + ex.Message);
```

Comment:
I suggest adding the ex to the inner exception and not append any additional exception message here.

Also pls. move the message and the creation of the exception to a factory method on the MsaClientlExeption class (or a factory or extension class)

Reply:
@jennyf19 added some additional comments. can be done in separate PR given this one has been closed.

Reply:
@henrik-me Sounds good. thanks